valispace/__init__.py

```diff
@@ -110,23 +110,199 @@ def get_vali(self, id=None, name=None):
 				print("VALISPACE-ERROR: Vali id must be an integer")
 				return
 
+		if workspace_id:
+			try:
+				workspace_id = int(workspace_id)
+			except:
+				print("VALISPACE-ERROR: Workspace id must be an integer")
+				return
+			
+		if project_id:
+			try:
+				project_id = int(project_id)
+			except:
+				print("VALISPACE-ERROR: Project id must be an integer")
+				return
+
+		if parent_id:
+			try:
+				parent_id = int(parent_id)
+			except:
+				print("VALISPACE-ERROR: Parent id must be an integer")
+				return
+
+		if tag_id:
+			try:
+				tag_id = int(tag_id)
+			except:
+				print("VALISPACE-ERROR: Tag id must be an integer")
+				return
+
+		if valis_marked_as_impacted:
```

Comment:
The naming of this parameter is confusing. Are you supposed to enter a list of valis? If so, this int() checking will always fail.

Reply:
Ya, it should be singular since it's just one id, not a list. Is "vali_marked_as_impacted" better?